docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.5'
+services:
+  web:
+    container_name: ruby-lang
+    image: ruby:2.6.1
```

Comment:
Maybe this can be updated? It seems that 2.6.4 is available: <https://hub.docker.com/_/ruby>. Currently .travis.yml specified the version to 2.6.3.

Reply:
@yous do I use the same version as travis or do I update Travis to use 2.6.4 as well?